src/createMatchEnhancer.js

```diff
@@ -31,9 +34,18 @@ export default function createMatchEnhancer(matcher) {
 
       const store = middlewareEnhancer(createStore)(...args);
 
+      function replaceRouteConfig(routeConfig) {
+        matcher.replaceRouteConfig(routeConfig);
+
+        store.dispatch({
+          type: FarceActionTypes.UPDATE_LOCATION,
+          payload: getFound(store.getState()).match.location,
```

Comment:
Is this kosher? Seems like the nicest way of getting the current location.

Reply:
i don't see why not